### PR TITLE
SCP SL TRANSLATION PL (March 9th) by Multiverse Uncle

### PR DESCRIPTION
**Pull request by MultiverseUncle**
# SCP:SL Translation PR Template

## Language
Polish (Polski)

## Type of change
- [ ] New addition
- [X] Changes to current existing

## Have you tested the changes you've made?
- [X] Yes
- [ ] No

# Checklist:

- [X] I have made sure to not commit changes to other stuff than my own or the default english one
- [X] I have ran `EncodingNormalizer.exe` on my changed files before commiting
- [ ] I have not made sure to not submit a duplicate translation language 
- [x] I have not included any advertisements/pointers to non-game stuff other than author in the folder name

### CODE REVIEW

File: Translations/Polish (Polski) by Multiverse Uncle/NewInventory.txt [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+Karta Woźnego:No description 
+Karta Naukowca:No description 
+Karta Inspektora Nadzorującego:No description 
+Karta Zarządcy Strefy:No description 
+Karta Ochroniarza:No description 
+Karta Kadeta MTF:No description 
+Karta Technika Zabezpieczeń:No description 
+Karta Porucznika MTF:No description 
+Karta Dowódcy MTF:No description 
+Karta Administratora Placówki:No description
```
Comment (MichalPetryka) [inline Translations/Polish (Polski) by Multiverse Uncle/NewInventory.txt]: Use encoding normalizer to get rid of those spaces

Comment (MultiverseUncle) [reply Translations/Polish (Polski) by Multiverse Uncle/NewInventory.txt]: Ok, done.

File: Translations/Polish (Polski) by Multiverse Uncle/MainMenu.txt [hunk 2]
```diff
@@ -26,7 +26,7 @@ Tryb renderowania świateł:\nKiedy wyłączone - pogarsza znacznie wygląd gry.
 V-SYNC
 POŁĄCZ
 USTAWIENIA JAKOŚCI
-Zmień preferencje cieni. Ustawienie tych suwaków przy niższych wartościach mogą zwiększyć płynność gry.
+Zmień preferencje cieni. Ustawienie tych suwaków przy niższych wartościach mogą zwiększyć płynność gry.
```
Comment (MichalPetryka) [inline Translations/Polish (Polski) by Multiverse Uncle/MainMenu.txt]: Also, fix this

Comment (MultiverseUncle) [reply Translations/Polish (Polski) by Multiverse Uncle/MainMenu.txt]: Okay

Comment (SandeQ) [reply Translations/Polish (Polski) by Multiverse Uncle/MainMenu.txt]: One more thing, "może zwiększyć płynnośc gry"

Comment (MultiverseUncle) [reply Translations/Polish (Polski) by Multiverse Uncle/MainMenu.txt]: Done

File: Translations/Polish (Polski) by Multiverse Uncle/Summary.txt [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
-KLASA D WYGRAŁA
+SCP WYGRYWA
+MTF WYGRYWA
+KLASA D WYGRYWA
 SYTUACJA PATOWA
 Są odpowiedzialne za śmierć [summary_scpfrags] istot ludzkich.
-Zagrożenie zostało usunięte. To miejsce jest już ponownie bezpieczne.
-Cała ochrona placówki zgineła.
+Usunięto zagrożenie. To miejsce jest już ponownie bezpieczne.
+Cała ochrona placówki została podana terminacji.
```
Comment (SandeQ) [inline Translations/Polish (Polski) by Multiverse Uncle/Summary.txt]:
```suggestion
Cała ochrona placówki została poddana terminacji.
```

File: Translations/Polish (Polski) by Multiverse Uncle/Summary.txt [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
 Obie strony poniosły straty jak i bardzo zyskały. Zwycięzca jest nieokreślony.
-[summary_d_escaped] klas D uciekło.
+[summary_d_escaped] klas D uciekło
 [summary_s_escaped] naukowców przetrwało
 Eksplozja głowicy Alpha pozbawiła życia [summary_warhead_kills] istot.
 Głowica Alpha nie została zdetonowana podczas misji.
 Misja zajęła [summary_round_minutes] minut i [summary_round_seconds] sekund.
 [summary_scp_terminated] unieszkodliwionych podmiotów SCP.
 następna runda zacznie się za [summary_next_round_countdown] sekund.
```
Comment (SandeQ) [inline Translations/Polish (Polski) by Multiverse Uncle/Summary.txt]:
```suggestion
Następna runda zacznie się za [summary_next_round_countdown] sekund.
```

File: Translations/Polish (Polski) by Multiverse Uncle/Class_Descriptions.txt [hunk 5]
```diff
@@ -1,17 +1,17 @@
-Zabij wszystkich\nNie pozwól nikomu uciec\nKooperuj z innymi <color=#f00>SCP</color><color=#ffff00>\nNamierz swoją ofiarę, i kliknij <color=#00ff00>lewy przycisk myszy</color> by jej złamać kark\nPoruszasz się wtedy, kiedy nikt na ciebie nie patrzy\nStajesz się szybszy, gdy masz mniej HP</color>
-Ucieknij z placówki\nWspółpracuj z <color=#008f1e>Rebelią Chaosu</color>\nBądź ostrożny
+Zabij wszystkich\nNie pozwól nikomu uciec\nKooperuj z innymi <color=#f00>SCP</color><color=#ffff00>\nNamierz swoją ofiarę, i kliknij <color=#00ff00>lewy przycisk myszy</color> by złamać jej kark\nPoruszasz się wtedy, kiedy nikt na ciebie nie patrzy\nStajesz się szybszy, gdy masz mniej HP</color>
+Ucieknij z placówki\nWspółpracuj z <color=#008f1e>Rebelią Chaosu</color>\nUnikaj zagrożenia
```
Comment (MichalPetryka) [inline Translations/Polish (Polski) by Multiverse Uncle/Class_Descriptions.txt]: Plural form, "zagrożeń" would be better.

File: Translations/Polish (Polski) by Multiverse Uncle/Class_Descriptions.txt [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
-Zabij wszystkich\nNie pozwól nikomu uciec\nKooperuj z innymi <color=#f00>SCP</color><color=#ffff00>\nNamierz swoją ofiarę i kliknij <color=#00ff00>lewy przycisk myszy</color>, by ją natychmiastowo zabić\nPrzytrzymaj <color=#00ff00>E</color> na osobie którą zabiłeś, by go przemienić w </color><color=#f00>SCP-049-2</color><color=#ffff00>\nTwoje przemienianie staje się szybsze, gdy masz mniej HP</color>
-Ucieknij z placówki\nWspółpracuj z <color=#0096FF>Nine-Tailed Fox</color>\nBądź ostrożny
+Pomóż <color=#FFFF7CFF>naukowcom</color> w ucieczce\nWyeliminuj zagrożenie\nInstruuj kadetów oraz ochroniarzy\nWykonuj rozkazy od Dowódcy
+Zabij wszystkich\nNie pozwól nikomu uciec\nKooperuj z innymi <color=#f00>SCP</color><color=#ffff00>\nNamierz swoją ofiarę i kliknij <color=#00ff00>lewy przycisk myszy</color>, by ją natychmiastowo zabić\nPrzytrzymaj <color=#00ff00>E</color> na osobie którą zabiłeś, aby przemienić ją w </color><color=#f00>SCP-049-2</color><color=#ffff00>\nTwoje przemienianie staje się tym szybsze, im masz mniej HP</color>
+Ucieknij z placówki\nWspółpracuj z <color=#0096FF>Nine-Tailed Fox</color>\nUnikaj zagrożenia
```
Comment (MichalPetryka) [inline Translations/Polish (Polski) by Multiverse Uncle/Class_Descriptions.txt]: Same

File: Translations/Polish (Polski) by Multiverse Uncle/Class_Descriptions.txt [hunk 7]
```diff
@@ -1,17 +1,17 @@
...
 Zabij wszystkich\nNie pozwól nikomu uciec\nKooperuj z innymi <color=#f00>SCP</color>
-Pomóż <color=#FF8E00>personelu klasy D</color> w ucieczce\nWyeliminuj każdą przeszkodę\nWykonaj swoją misję szybko i efektywnie.
-Zabij wszystkich\nNie pozwól nikomu uciec\nKooperuj z innymi <color=#f00>SCP</color><color=#ffff00>\nJesteś nieśmiały\nWpadasz w furię kiedy jakaś osoba cię na ciebie spojrzy\nPodczas furii jesteś o wiele szybszy, wywarzasz drzwi oraz <color=#00ff00>lewym przyciskiem myszy</color> zabijasz każdego kogo spotkasz\nPo furii się na chwilę uspokajasz</color>
+Pomóż <color=#FF8E00>personelowi klasy D</color> w ucieczce\nWyeliminuj każdą przeszkodę
+Zabij wszystkich\nNie pozwól nikomu uciec\nKooperuj z innymi <color=#f00>SCP</color><color=#ffff00>\nJesteś nieśmiały\nWpadasz w furię, kiedy jakaś osoba się na ciebie spojrzy\nPodczas furii jesteś o wiele szybszy, wyważasz drzwi oraz <color=#00ff00>lewym przyciskiem myszy</color> zabijasz każdego kogo spotkasz\nPo furii się na chwilę uspokajasz</color>
```
Comment (MichalPetryka) [inline Translations/Polish (Polski) by Multiverse Uncle/Class_Descriptions.txt]: "gdy ktoś się na ciebie spojrzy" would sound better

File: Translations/Polish (Polski) by Multiverse Uncle/Connection_Errors.txt [hunk 8]
```diff
@@ -1,9 +1,9 @@
-<size=80>ZOSTAŁEŚ WYRZUCONY</size>\nSerwer wymusił zatrzymanie połączenia.\n<size=26>\nZostałeś prawdopodobnie czasowo zablokowany przez administratora\n lub nie spełniasz wymagań serwera (np. wersja gry).\nZa ewentualne blokady autorzy gry <u>nie</u> ponoszą odpowiedzialności.\n</size>
+<size=80>ZOSTAŁEŚ WYRZUCONY</size>\nSerwer wymusił zatrzymanie połączenia.\n<size=26>\nZostałeś prawdopodobnie czasowo zablokowany przez administratora\n lub nie spełniasz wymagań serwera (np. wersja gry).\nZa ewentualne blokady autorzy gry <b>nie</b> ponoszą odpowiedzialności.\n</size>
```
Comment (MichalPetryka) [inline Translations/Polish (Polski) by Multiverse Uncle/Connection_Errors.txt]: "Serwer wymusił rozłączenie" would be better

File: Translations/Polish (Polski) by Multiverse Uncle/NewInventory.txt [hunk 9]
```diff
@@ -20,12 +20,12 @@ Kubek:Zrobiony z plastiku.
...
 Logicer:No description
 Granat Odłamkowy:Kaboom!
-Granat Odblaskowy:Oślepia przeciwnika.
+Granat Błyskowy:Oślepia przeciwnika.
 Rozbrajacz:Lewym rozbrajasz, prawym uwalniasz. \nPrzytrzymaj E na kimś kto jest rozbrojony w twojej drużynie, by go uwolnić.
 Upuszczone 7.62:Debug Item.
-Dropped 9mm:No description.
-USP:No description.
\ No newline at end of file
+Upuszczone 9mm:Spolszczona wersja\n8.0.1
```
Comment (MichalPetryka) [inline Translations/Polish (Polski) by Multiverse Uncle/NewInventory.txt]: "Spolszczona wersja\n8.0.1"?

Comment (MultiverseUncle) [reply Translations/Polish (Polski) by Multiverse Uncle/NewInventory.txt]: This was temporarily for old translation upload method. Deleted now.

**Comment by MichalPetryka** [comment]
Now everything seems ok, i think

**Pull request merged**